sympy/series/limits.py

```diff
@@ -136,6 +136,11 @@ def doit(self, **hints):
         if not e.has(z):
             return e
 
+        # limit can work with binomial by converting to factorial first
+        if (e.has(binomial) and not (hints.get('sequence', True)
```

Comment:
This appears a little hacky. I would go with implementing ``_eval_as_tractable``.

```
>>> e = 16**k / (k * binomial(2*k, k)**2)
>>> limit(e.rewrite(gamma), k, oo) # works
pi
```
By applying the patch,

```
diff --git a/sympy/functions/combinatorial/factorials.py b/sympy/functions/combinatorial/factorials.py
index 48ea175..0b7adb2 100644
--- a/sympy/functions/combinatorial/factorials.py
+++ b/sympy/functions/combinatorial/factorials.py
@@ -853,6 +853,8 @@ def _eval_rewrite_as_gamma(self, n, k):
         from sympy import gamma
         return gamma(n + 1)/(gamma(k + 1)*gamma(n - k + 1))
 
+    _eval_rewrite_as_tractable = _eval_rewrite_as_gamma
+
     def _eval_rewrite_as_FallingFactorial(self, n, k):
         if k.is_integer:
             return ff(n, k) / factorial(k)
```

```
>>> limit(e.rewrite(gamma), k, oo)
pi
>>> limit(e, k, oo) # This seems odd
oo
```

ping @asmeurer